packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsEditBoxPicker.tsx

```diff
@@ -46,14 +52,18 @@ export const ChatMessageComponentAsEditBoxPicker = (props: ChatMessageComponentA
   /* @conditional-compile-remove(rich-text-editor) */
   const { richTextEditor } = props;
 
+  const simpleEditBox = useMemo(() => {
+    return <ChatMessageComponentAsEditBox {...props} />;
+  }, [props]);
+
   /* @conditional-compile-remove(rich-text-editor) */
   if (richTextEditor) {
     return (
-      <Suspense>
+      <Suspense fallback={simpleEditBox}>
```

Comment:
I was thinking may also need to use an error boundary here in the event that the bundle fails to load.

https://legacy.reactjs.org/docs/code-splitting.html#error-boundaries

Where the error boundary would be the fall back if everything fails and loading spinner would be the the suspense fallback in unlikely event that it is slow to load even with the preload we are now doing.

Reply:
Good catch, I'll add it (didn't know about the error boundaries)

Reply:
Thanks!